tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -260,63 +235,76 @@ public function testListItem(): void {
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--thumbnail'));
```

Comment:
We should keep this assertion.

Reply:
Already present on the block for the image. check below.